cmd/skaffold/app/cmd/inspect_modules.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
```

Comment:
do we want to move this into a subpackage? A bit worried that we may clutter up the `cmd/` dir even more with these subcommands

Reply:
I took a stab at it but it's very convoluted the way `command builder`, `command definitions` and `command instantiations` interact along with common and specific `flags` instantiations. I'll look at that separate from this PR.